### ISSUE TITLE
indent size is 2 instead of 4 - Failed to resolve config - No local configuration detected

### ISSUE THREAD

**Issue by wassim-azirar** [prettier/prettier-vscode#1082]
## Summary
My .prettierrc file is not used to format my files.

### Expected result

I expect the file to be formatted with 4 spaces

### Actual result

The file gets formatted with 2 spaces

 ### Screenshot
![image](https://user-images.githubusercontent.com/670586/69247299-7d22cd00-0baa-11ea-9b20-6be661443e40.png)

**VS Code Version**: 1.40.1

**Prettier & Prettier Plugin Version**:
"prettier": "^1.19.1"
"prettier plugin": [3.8.0]

**OS and version**:
Windows 10 Pro

## .editorconfig
```
root = true

[*]
charset = utf-8
indent_style = space
indent_size = 4
end_of_line = lf
insert_final_newline = true
trim_trailing_whitespace = true
```

## .prettierrc
```
{
    "bracketSpacing": true,
    "bracesSpacing": true,
    "jsxBracketSameLine": true,
    "jsxSingleQuote": true,
    "noSemi": true,
    "printWidth": 120,
    "singleQuote": true,
    "tabWidth": 4,
    "trailingComma": "none",
    "useTabs": false
}
```

## .prettierignore
```
.vscode/*
package.json
yarn.lock
```

I tried to delete the .prettierignore file, but I keep seeing this log :

**Resolved ignore file to c:\repos\project\\.prettierignore**

I restared VSCode many times, and everytime I use the format commande I see this message. :angry:

## settings.json
```json
{
  "editor.tabSize": 4,
  "editor.detectIndentation": false,
  "editor.defaultFormatter": "esbenp.prettier-vscode",
  "[javascriptreact]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode"
  },
  "[javascript]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode"
  }
}
```

## Prettier Log Output
```
[INFO - 3:14:35 PM] Formatting c:\repos\project\Components\Order.jsx.
[INFO - 3:14:35 PM] Loaded module 'prettier@1.19.1' from 'c:\repos\project\node_modules\prettier\index.js'.
[INFO - 3:14:35 PM] Resolved ignore file to c:\repos\project\.prettierignore.
[INFO - 3:14:35 PM] File Info:
{
  "ignored": false,
  "inferredParser": "babel"
}
[ERROR - 3:14:35 PM] Failed to resolve config for c:\repos\project\Components\Order.jsx. Falling back to the default config settings.
[INFO - 3:14:35 PM] No local configuration detected, using VS Code configuration.
[INFO - 3:14:35 PM] Prettier Options:
{
  "arrowParens": "avoid",
  "bracketSpacing": true,
  "endOfLine": "auto",
  "htmlWhitespaceSensitivity": "css",
  "insertPragma": false,
  "jsxBracketSameLine": false,
  "jsxSingleQuote": false,
  "printWidth": 80,
  "proseWrap": "preserve",
  "quoteProps": "as-needed",
  "requirePragma": false,
  "semi": true,
  "singleQuote": false,
  "tabWidth": 2,
  "trailingComma": "none",
  "useTabs": false,
  "vueIndentScriptAndStyle": false,
  "filepath": "c:\\repos\\project\\Components\\Order.jsx",
  "parser": "babel"
}
[INFO - 3:14:35 PM] Formatting completed in 31.861999ms.
```

**Comment by ntotten** [comment]
> Resolved ignore file to c:\repos\project\.prettierignore 
This doesn't mean anything, it just means that is the location that it looks for the ignore file. If the file is not there, then it isn't used.

Is your `.prettierrc` file located at `c:\repos\project\.prettierrc`? What about your `.editorconfig`, where is that located?

This extension uses prettier to resolve configuration files which is: 

"The configuration file will be resolved starting from the location of the file being formatted, and searching up the file tree until a config file is (or isn't) found."

It sounds like your configuration files are not in the same tree as the file being formatted so they wouldnt be discovered.

**Comment by wassim-azirar** [comment]
@ntotten my **.prettierrc** is located at the root folder `c:\repos\project\`
I also tried to move **.prettierrc** at the same level of my components `c:\repos\project\Components\`

But I still get that error `Failed to resolve config for c:\repos\project\Components\Shopping\Order.jsx. Falling back to the default config settings.`

To be sure that EditorConfig is not causing any conflicts. I removed **.editorconfig** from my root folder but It still doesn't work.

**Comment by anvinjain** [comment]
Seeing the same problem as @wassim-azirar 
I have .prettierrc.yaml in repository root but settings inside that are not being respected when vs code runs prettier.

**Comment by ntotten** [comment]
I think I have somewhat figured out what is going on here. It appears to be a text encoding issue with the `.prettierrc` file. I added some additional logging. When using the provided repo I noticed the following error:

![image](https://user-images.githubusercontent.com/282782/69724042-f65d7b00-10e8-11ea-95a2-c10811d885a1.png)

The immediate fix is to change the encoding of the `.prettierrc` file to utf8. I will try to see if there is a better fix later, but this should unblock you at the moment.
